assets/prototype/domain/eventEditor/context/DateTimeProvider/useDateEditorId.ts

```diff
@@ -7,7 +7,7 @@ import { EditorId } from '../../../../application/ui/components/layout/editor-mo
 /**
  * Function that returns editor id for a given event date
  */
-const useDateEditorId = (prefix: string = 'datetime-editor', datetimeId: EntityId): EditorId => {
+const useDateEditorId = (prefix = 'datetime-editor', datetimeId: EntityId): EditorId => {
 	return prefix !== '' ? `${prefix}-${datetimeId}` : `datetime-editor-${datetimeId}`;
 };
```

Comment:
Fix for the:
![14_jan_1](https://user-images.githubusercontent.com/1477453/72341473-0817d380-36d3-11ea-83a5-a24277d68fb8.png)